version/version.go

```diff
@@ -7,7 +7,7 @@ import (
 const (
 	VersionMajor = 2  // Major version component of the current release
 	VersionMinor = 0  // Minor version component of the current release
-	VersionPatch = 1  // Patch version component of the current release
+	VersionPatch = 2  // Patch version component of the current release
 	VersionMeta  = "" // Version metadata to append to the version string
```

Comment:
I would suggest to set the version meta to "dev" for the development and main branch, to differentiate between the actual "v2.0.2" release and development builds.